packages/workspace/src/utils/graph-utils.ts

```diff
@@ -5,7 +5,7 @@ import {
 import { isWorkspaceProject } from '../core/project-graph/operators';
 
 interface Reach {
-  graph: ProjectGraph;
+  graph: ProjectGraph<any, any>;
```

Comment:
Why is this necessary? Why don't default type values work?

Reply:
Default second generic type falls back to `FileData[]` which is then not type-compatible with `Record<string, FileData>`.